package.json

```diff
@@ -1,109 +1,109 @@
-        "start": "remix-serve build",
-        "deploy": "fly deploy --remote-only",
-        "dev": "remix dev",
-        "jest-preview": "jest-preview",
-        "typecheck": "tsc",
-        "lint": "eslint --ignore-path .gitignore '**/*.{js,jsx,ts,tsx}'",
-        "lint:fix": "eslint --ignore-path .gitignore '**/*.{js,jsx,ts,tsx}' --fix",
-        "test": "jest",
-        "test:watch": "jest --watch",
-        "test:ci": "jest --ci",
-        "test:e2e": "playwright test",
-        "test:e2e:ui": "playwright test --ui",
-        "test:e2e:report": "playwright show-report",
-        "test:preview": "npm-run-all -p test jest-preview"
-    },
-    "dependencies": {
-        "@remix-run/css-bundle": "1.19.3",
-        "@remix-run/node": "1.19.3",
-        "@remix-run/react": "1.19.3",
-        "@remix-run/serve": "1.19.3",
-        "@sentry/remix": "^7.68.0",
-        "bignumber.js": "^9.1.1",
-        "bootstrap": "^5.3.1",
-        "express-rate-limit": "^7.0.0",
-        "isbot": "^3.6.13",
-        "js-xdr": "3.0.0",
-        "lodash": "^4.17.21",
-        "prismjs": "^1.29.0",
-        "react": "^18.2.0",
-        "react-bootstrap": "2.8.0",
-        "react-dom": "^18.2.0",
-        "react-intl": "6.4.4",
-        "react-json-pretty": "^2.2.0",
-        "react-router-bootstrap": "^0.26.2",
-        "remix-utils": "^6.6.0",
-        "stellar-sdk": "11.0.1",
-        "urijs": "1.19.11"
-    },
-    "devDependencies": {
-        "@flydotio/dockerfile": "^0.4.7",
-        "@playwright/test": "^1.40.0",
-        "@remix-run/dev": "1.19.3",
-        "@remix-run/eslint-config": "1.19.3",
-        "@remix-run/testing": "^2.3.1",
-        "@swc/core": "^1.3.99",
-        "@swc/jest": "^0.2.29",
-        "@testing-library/jest-dom": "^6.1.4",
-        "@testing-library/react": "^14.1.2",
-        "@testing-library/user-event": "^14.5.1",
-        "@types/jest": "^29.5.10",
-        "@types/lodash": "^4.14.196",
-        "@types/node": "^20.10.0",
-        "@types/prismjs": "^1.26.0",
-        "@types/react": "^18.2.17",
-        "@types/react-bootstrap": "^0.32.32",
-        "@types/react-copy-to-clipboard": "^5.0.4",
-        "@types/react-dom": "^18.2.7",
-        "@types/react-router-bootstrap": "^0.26.2",
-        "@types/urijs": "^1.19.25",
-        "@typescript-eslint/parser": "^5.62.0",
-        "eslint": "^8.46.0",
-        "eslint-config-prettier": "^9.0.0",
-        "eslint-config-react-app": "^7.0.1",
-        "eslint-plugin-prettier": "^5.0.1",
-        "husky": "^8.0.3",
-        "jest": "^29.7.0",
-        "jest-environment-jsdom": "^29.7.0",
-        "jest-preview": "^0.3.1",
-        "lint-staged": "^15.1.0",
-        "npm-run-all": "^4.1.5",
-        "ts-node": "^10.9.1",
-        "typescript": "^5.1.6"
-    },
-    "husky": {
-        "hooks": {
-            "pre-commit": "lint-staged"
-        }
-    },
-    "lint-staged": {
-        "*.{js,jsx,ts,tsx}": "eslint --ignore-path .gitignore"
+  "name": "stellarexplorer",
```

Comment:
[note]

As I wrote below, the format of package.json was automatically fixed to indent 2.
https://github.com/chatch/stellarexplorer/pull/550#issue-2007549129

But it seemed to be changed to indent 4 (on master).
I changed it to 2 again.
But I don't have a strong opinion on that.

Reply:
i think 2 is consistent with the ts/js and i generally prefer it so it's fine.